backend/src/models/encounter.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+
+interface EncounterModel {
+  date: Date,
+  location: string,
+  description: string,
+  persons: [Schema.Types.ObjectId]
+}
+
+const schema = new Schema<EncounterModel>({
+  date: { type: Date, default: Date.now },
+  location: { type: String, required: false },
+  description: { type: String, required: true },
+  persons: { type: [Schema.Types.ObjectId], required: true }
```

Comment:
does making the `persons` of type `Schema.Types.ObjectId` mean it's storing the whole Person object or is it just a reference to a person object's ID?

Reply:
Just the reference ID.

Reply:
Sweet, that makes sense.